CHANGES.rst

```diff
@@ -84,7 +84,8 @@ New Features
 
   - Added ``NDDataRef`` that implements ``NDData`` together with all
     currently avaiable mixins. This class does not implement additional
-    attributes, methods or a numpy.ndarray-like interface like ``NDDataArray``.
+    attributes, methods or a numpy.ndarray-like interface like ``NDDataArray``. 
```

Comment:
Found a trailing whitspace in this line. And thank you for including the PR number. :+1: